### PR TITLE
Adds a preferences toggle for VOX speech

### PR DESCRIPTION
**Pull request by Absolucy**
:cl: Lucy
tweak: You can now disable hearing AI VOX speech!
/:cl:

### CODE REVIEW

**Comment by SomeguyManperson** [comment]
switchover switchover switchover switchover switchover switchover switchover switchover switchover switchover switchover switchover switchover switchover switchover switchover

**Comment by yacabo** [comment]
You know VOX is often used to relay critical messages right? I don't think it's a good idea to let players flat out be able to miss very important announcements

**Comment by Absolucy** [comment]
@yacabo I dunno about often, usually it's used for "THIS. STATION. IS. NOW. UNDER. MILITARY. COMMAND."

File: code/__DEFINES/{yogs_defines}/preferences.dm [hunk 1]
```diff
@@ -8,10 +8,15 @@
 
 #define GHOST_CKEY	(1<<11)
 
+#define SOUND_VOX	(1<<22)
```
Comment (JamieD1) [inline code/__DEFINES/{yogs_defines}/preferences.dm]: Does this need to be 22?

File: code/__DEFINES/{yogs_defines}/preferences.dm [hunk 2]
```diff
@@ -8,10 +8,15 @@
...
 
 //YOGS pref.yogstoggles enum's
 
 #define QUIET_ROUND				(1<<0) //Donor features, quiet round; in /~yogs_defines/, as god intended
 #define PREF_MOOD				(1<<1) //Toggles the use of the Mood feature. Defaults to off, thank god.
 
 #define YOGTOGGLES_DEFAULT 0
+
+#undef TOGGLES_DEFAULT
+#define TOGGLES_DEFAULT (SOUND_ADMINHELP|SOUND_MIDI|SOUND_AMBIENCE|SOUND_LOBBY|MEMBER_PUBLIC|INTENT_STYLE|MIDROUND_ANTAG|SOUND_INSTRUMENTS|SOUND_SHIP_AMBIENCE|SOUND_PRAYERS|SOUND_ANNOUNCEMENTS|SOUND_JUKEBOX|SOUND_VOX)
```
Comment (JamieD1) [inline code/__DEFINES/{yogs_defines}/preferences.dm]: Why do we need to change this?

File: code/modules/client/preferences_savefile.dm [hunk 3]
```diff
@@ -5,7 +5,7 @@
 //	You do not need to raise this if you are adding new values that have sane defaults.
 //	Only raise this value when changing the meaning/format/name/layout of an existing value
 //	where you would want the updater procs below to run
-#define SAVEFILE_VERSION_MAX	33
+#define SAVEFILE_VERSION_MAX	34 // yogs edit -- bump savefile verison
```
Comment (JamieD1) [inline code/modules/client/preferences_savefile.dm]: Do you need to add a new version if its an addition?

**Review by JamieD1 — CHANGES_REQUESTED** [review]

File: code/__DEFINES/{yogs_defines}/preferences.dm (hunk 2 above)
Comment (Absolucy) [reply code/__DEFINES/{yogs_defines}/preferences.dm]: So VOX is on by default

File: code/modules/client/preferences_savefile.dm (hunk 3 above)
Comment (Absolucy) [reply code/modules/client/preferences_savefile.dm]: Not sure - this just enables VOX by default in all old savefiles

File: code/__DEFINES/{yogs_defines}/preferences.dm (hunk 1 above)
Comment (Absolucy) [reply code/__DEFINES/{yogs_defines}/preferences.dm]: https://github.com/yogstation13/Yogstation/blob/28af47f93de3a371b1cfdb5d634801de684a20f9/code/__DEFINES/preferences.dm#L25

**Review by Bibby0110 — COMMENTED** [review]
please for the love of god un-modularize this.
we don't do modularization anymore and this is super confusing for any new people coming to change preferences

**Comment by Absolucy** [comment]
ah, okay. should we unmodularize existing modularized things at some point?

**Comment by Bibby0110** [comment]
> ah, okay. should we unmodularize existing modularized things at some point?

We should preferably just do it as we come across it, people didn’t like doing everything at once for some reason

**Comment by Absolucy** [comment]
@TheGamerdk alright, rewritten. someone please merge because I just heard an AI spam the honk vox and I want to stab my eardrums.

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**